website/frontend/management/commands/scraper.py

```diff
@@ -256,7 +281,9 @@ def add_to_git_repo(data, filename, article):
         my_hash = run_git_command(['hash-object', filename],
                                   article.full_git_dir).strip()
 
+        # This is going to look this up separately each time
         commits = [v.v for v in article.versions()]
+        # Why > 2?  Why not > 1?
```

Comment:
If there are only two commits, then they can't be different.  This branch is to detect when an article is bouncing back and forth between two versions (probably because one server has an outdated copy) and not create infinitely many versions.